internal/proxy/task_scheduler.go

```diff
@@ -372,9 +372,9 @@ func newDqTaskQueue(tsoAllocatorIns tsoAllocator, idAllocatorIns idAllocatorInte
 }
 
 type taskScheduler struct {
-	ddQueue taskQueue
+	ddQueue *ddTaskQueue
```

Comment:
please do not modify this.

Reply:
I don't know why we can't modify this. It's just a struct using internal the proxy.

Reply:
If i don't modify this, it will introduce big complexity when I have to cover test cases that task faile to enqueue.